contracts/utils/BaalVotes.sol

```diff
@@ -0,0 +1,235 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
```

Comment:
lets make these consistent and remove the carrot here

pragma solidity 0.8.13;